interop/tapir/src/test/scala/caliban/interop/tapir/TestService.scala

```diff
@@ -89,6 +94,8 @@ object TestService {
 
       def deletedEvents: ZStream[Any, Nothing, String] =
         ZStream.unwrapManaged(subscribers.subscribe.map(ZStream.fromQueue(_)))
+
+      override def reset: UIO[Unit] = characters.set(initial)
```

Comment:
```suggestion
      def reset: UIO[Unit] = characters.set(initial)
```
just for consistency